client-js/app/elements/labrad-registry.html

```diff
@@ -8,15 +8,24 @@
     :host {
       display: block;
     }
-    #reg-menu {
+
+    .z-container {
+      background: white;
+      height: 100%;
+      margin: 0px;
+    }
+
+    .reg-menu {
       position: fixed;
-       @apply(--shadow-elevation-3dp);
-       padding-top: 5px;
-       padding-bottom: 5px;
-       background-color: white;
-       z-index: +1;
+      top: 140px;
+      left: 33px;
```

Comment:
Yeah, I agree. I'm sure there's a correct way to do this, I just don't know it. Things like this I can keep in mind and fix when I learn it.